src/Microsoft.AspNetCore.Hosting/WebHostBuilderExtensions.cs

```diff
@@ -141,8 +131,34 @@ public static IWebHostBuilder ConfigureLogging(this IWebHostBuilder hostBuilder,
         /// <param name="configureLogging">The delegate that configures the <see cref="LoggerFactory"/>.</param>
         /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
         public static IWebHostBuilder ConfigureLogging(this IWebHostBuilder hostBuilder, Action<WebHostBuilderContext, ILoggingBuilder> configureLogging)
-        {
-            return hostBuilder.ConfigureServices((context, collection) => collection.AddLogging(builder => configureLogging(context, builder)));
-        }
+            => hostBuilder.ConfigureServices((context, collection) => collection.AddLogging(builder => configureLogging(context, builder)));
+
+        /// <summary>
+        /// Registers an object that will have all of its I[Post]ConfigureOptions registered.
+        /// </summary>
+        /// <param name="hostBuilder">The <see cref="IWebHostBuilder" /> to configure.</param>
+        /// <param name="configureOptionsInstance">Object that will have all of its I[Post]ConfigureOptions registered.</param>
+        /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
+        public static IWebHostBuilder ConfigureOptions(this IWebHostBuilder hostBuilder, object configureOptionsInstance)
```

Comment:
Why do we need these?

Reply:
Guess we don't, so are we okay with just having the new ConfigureOptions methods, and no new sugar needs to be added to Hosting for https://github.com/aspnet/Options/issues/224?

I can still add a sample/tests to hosting if we want, but if there's no new APIs, its mostly just as an example